requirements/extras/confluentkafka.txt

```diff
@@ -1 +1 @@
-confluent-kafka>=2.2.0
+confluent-kafka>=2.2.0 ; python_version < "3.13"
```

Comment:
This line is necessary for the installation of requirements. The change should be removed before the merge.

https://github.com/confluentinc/confluent-kafka-python/issues/1768